pages/help/help_faq.md

```diff
@@ -100,3 +105,26 @@ If you’re unsure who your DNS hosting provider is, sign in to the .gov registr
 **A .gov domain cannot be transferred to any other registrar**. However, DNS for the domain can be managed by a third party, and as some DNS hosting providers operate registrars, their help desk or documentation may not clearly reflect the unique case of .gov domains.
 
 If you are asked for an AUTH code by your DNS hosting provider, let them know that you’re trying to set up DNS hosting and that you cannot transfer a .gov domain to them.
+
+<span id="commercial-purposes"></span>
+## Your policies prohibit using a .gov domain for “commercial purposes”. What does that mean?
+The [DOTGOV Act of 2020](https://uscode.house.gov/view.xhtml?req=(title:6%20section:665%20edition:prelim)%20OR%20(granuleid:USC-prelim-title6-section665)&f=treesort&edition=prelim&num=0&jumpTo=true){.usa-link--external} states that .gov domains “may not be used for commercial… purposes.” This means a .gov domain may not be used for purposes that would solely promote a commercial entity or their activities. For example, a .gov website should not include promotional space that can be directly purchased (e.g., web advertising). However, a .gov domain can be used in ways that promote commerce where such promotion primarily serves a public purpose rather than private interests. As appropriate, this might include the names or logos of private enterprises on a .gov website. 
+
+Not all government activities that involve money constitute “commercial purposes”. For example, a state Department of Motor Vehicles could use a .gov domain to collect payment for license renewals. Payment for government services, or for the entity’s own fundraising (as appropriate), would not constitute “commercial” purposes. 
```

Comment:
```suggestion
Not all government activities that involve money constitute “commercial purposes." For example, a state Department of Motor Vehicles could use a .gov domain to collect payment for license renewals. Payment for government services, or for the entity’s own fundraising (as appropriate), would not constitute “commercial purposes." 
```

Reply:
I adjusted the quotation marks around "commercial purposes" ... but I don't think they're really necessary in this paragraph. I think it's correct with or without the quotations, but including them does make those words stand out more, which may be the goal.